tokenizers/src/normalizers/byte_level.rs

```diff
@@ -0,0 +1,180 @@
+use crate::processors::byte_level::bytes_char;
+use crate::tokenizer::{NormalizedString, Normalizer, Result};
+use serde::{Deserialize, Serialize};
+use std::collections::{HashMap, HashSet};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(tag = "type")]
+pub struct ByteLevel {}
+
+lazy_static! {
+    static ref BYTES_CHAR: HashMap<u8, char> = bytes_char();
+    static ref CHAR_BYTES: HashMap<char, u8> =
+        bytes_char().into_iter().map(|(c, b)| (b, c)).collect();
+}
+
+impl Default for ByteLevel {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl ByteLevel {
+    pub fn new() -> Self {
+        Self {}
+    }
+
+    pub fn alphabet() -> HashSet<char> {
+        BYTES_CHAR.values().copied().collect()
+    }
+}
+
+impl Normalizer for ByteLevel {
+    /// Strip the normalized string inplace
+    fn normalize(&self, normalized: &mut NormalizedString) -> Result<()> {
```

Comment:
Why doesn't this live in `NormalizedString` like so:
```rs
impl NormalizedString {
  fn normalize_byte_level(&mut self) -> Result<()> { // ... }
}
```
?
Feels a bit weird to have an empty stateless struct for just a function, but may be due to the structure of tokenizers / python.